Orange/tests/test_classification.py

```diff
@@ -209,6 +210,13 @@ def test_continuous(lf):
         assert(all(tree(strlist) == tree(Orange.data.Table(table.domain, strlist))))
 
 
+class UnknownValuesInPrediction(unittest.TestCase):
+    def test_unknown(self):
+        table = Table("iris")
+        tree = Orange.classification.TreeLearner()(table)
+        tree([1, 2, None])
```

Comment:
This test is pointless - it passes without changes. Maybe you could replace TreeLearner with LogisticRegressionLearner.